spec/TodoSpec.js

```diff
@@ -5,10 +5,12 @@ let number2 = 5;
 
 var calculationPage = new CalculationPage();
 var calculationHelper = new CalculationHelper();
+beforeEach(function() {
```

Comment:
With the current application, this is ideal but with CRM application we may need a function which will construct the URL if needed with multiple parameters